.gitlab-ci.yml

```diff
@@ -52,6 +54,8 @@ before_script:
     - ls
     - echo ${PWD}
     - echo "${STARTUP_SCRIPT}"
+    # Docker is required on the ansible host for download_run_once and download_localhost
+    - docker info
```

Comment:
This will make Packet jobs fail that does not have dind, and there is no need to print out docker info, so you can skip this.

Reply:
@woopstar you're right. I checked the runner ConfigMap of the Packet runner, it doesn't have `privileged = true` which is required for dind to work. 

I'm thinking to do `download_localhost: false` for all Packet jobs instead since privileged containers are not desirable.